### PR TITLE
docs: add typedoc documentation to `client-react-hooks`

### PR DESCRIPTION
**Pull request by cyberglot**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,14 +42,15 @@
     "lint-staged": "^15.2.9",
     "node-fetch": "^3.3.2",
     "prettier": "^3.3.3",
-    "serve": "^14.2.3",
+    "serve": "^6.5.8",
```
Comment (tim-hm) [inline package.json:45]: This doesn't seem right ... 6.5.8 is a few years old?

File: packages/client-react-hooks/src/use-nil-compute.ts [hunk 2]
```diff
@@ -10,15 +10,31 @@ import {
 import { nilHookBaseResult, UseNilHook } from "./nil-hook-base";
 import { useNillion } from "./use-nillion";
 
-interface ExecuteArgs {
+/** ExecuteArgs is an interface that can be passed to the `execute` function.
+ * @param bindings: `ProgramBindings`
+ * @param values?: `NadaValues`
+ * @param storeIds?: array of `StoreId`s or strings
+ */
+export interface ExecuteArgs {
```
Comment (tim-hm) [inline packages/client-react-hooks/src/use-nil-compute.ts:18]: Exporting this without adjustment creates type collisions. Shall we rename these after their hooks, eg, `UseNilComputeExecuteArgs`?

File: typedoc.json [hunk 3]
```diff
@@ -2,8 +2,8 @@
   "$schema": "https://typedoc.org/schema.json",
   "cleanOutputDir": true,
   "entryPoints": [
-    "packages/client-core/src/index.ts",
-    "packages/client-payments/src/index.ts",
+    // "packages/client-core/src/index.ts",
```
Comment (tim-hm) [inline typedoc.json:5]: Are these commented out accidentally or deliberately? If deliberately, then removing the dead code altogether, with an comment in the commit message, is preferred.

**Review by tim-hm — CHANGES_REQUESTED** [review]
A few points for discussuion.

1. @oceans404 do we have a documentation style (i.e. tense, etc)?

For example:

```
`UseNilCompute` is a hook that allows you to execute a compute operation on Nillion.
```

or

```
The `UseNilCompute` hook executes a compute operation on Nillion.
```

or

something else entirely?

2. I've started a thread in the toolchain team channel to discuss if we should keep the generated html pages.

File: package.json (hunk 1 above)
Comment (cyberglot) [reply package.json:45]: Interesting, I'm assuming npm audit "updated" this 🤔 Fixed!

File: typedoc.json (hunk 3 above)
Comment (cyberglot) [reply typedoc.json]: I left them bc they would be needed back eventually, but it doesn't matter anymore.

**Comment by crypblizz8** [comment]
Closing due to difference in SDK state now

**Pull request closed without merging**